doc/website/getting-started/overview.md

```diff
@@ -627,28 +634,14 @@ Now that we have applications capable of sending and receiving data, we can run
 
 First we need to start RouDi.
 
-    # If installed and available in PATH environment variable
-    iox-roudi
-    # If build from scratch with script in tools
-    $ICEORYX_ROOT/build/posh/iox-roudi
+```
+# If installed and available in PATH environment variable
+iox-roudi
+# If build from scratch with script in tools
+$ICEORYX_ROOT/build/posh/iox-roudi
+```
 
 Afterwards we can start the applications which immediately connect to the RouDi via their runtime.
 
 When the application terminates, the runtime cleans up all resources needed for communication with RouDi. This
 includes all memory chunks used for the data transmission which may still be hold by the application.
-
```

Comment:
Have you moved the following chapters to another file? I prefer to not just delete them.

Reply:
The chapters didn't quite fit into the structure if you look at it overall:

![screenshot-iceoryx-io](https://user-images.githubusercontent.com/8661268/103881752-2c867000-50db-11eb-99f1-05f525ac6e57.png)

What do you think about removing "Exanples and adding an introduction chapter after "Using the API" that explains that there is a C and C++ API?

Reply:
The C API is already mentioned in the introduction of the "API" chapter. Maybe you can add the hint in line 654 (or move 654 to 121) and then remove the "C API" chapter.
If you want to remove the "Examples" chapter then please add at least a link to the examples readme in the introduction of the "Using the API" chapter.

Reply:
Added a explanation and a link to the C example.

Link to the examples page is already there, see third paragraph in _General_ on the screenshot.